### ISSUE TITLE
Use few task names in single run

### ISSUE THREAD

**Issue by ophir91** [allegroai/trains#18]
Hello, 
First Thanks for this project it looks very helpful.

I like to run a cross validation loop and save the results under the same project name but with different task name.

There is a way to tell trains that i now i finish one task and start another? 
When i just try to do the Task.init() again with the same project name and different task name i got an error:

```
Traceback (most recent call last):
  File "/media/ophir/DATA1/***/***/***/***/main.py", line 84, in <module>
    descrip = ''
  File "/media/ophir/DATA1/software/anaconda3/envs/py36/lib/python3.6/site-packages/trains/task.py", line 167, in init
    verify_defaults_match()
  File "/media/ophir/DATA1/software/anaconda3/envs/py36/lib/python3.6/site-packages/trains/task.py", line 161, in verify_defaults_match
    current=current,
trains.errors.UsageError: Current task already created and requested task name 'validIter1' does not match current task name 'validIter0'   
```

Thanks
Ophir

**Comment by bmartinn** [comment]
@ophir91 I assume you want a single script to create two experiments one after the other.
For example train and test as separate experiments in the system.
I would suggest the following:
```
task = Task.init('example', 'train')
# train here
task.close()

task - Task.init('example', 'train')
# test model here
task.close()
```

Let me know if that what you were looking for.

**Comment by ophir91** [comment]
Hey, 
Thanks for the comment
This is looks the same as i want. 
So i tried it.
But, I got the log file with all the experiments outputs, only on the first experiment.
The rest of them are without log.

do you know why?

Thanks.
Ophir

**Comment by bmartinn** [comment]
Sounds like a bug :)
Let me check...

**Comment by ophir91** [comment]
Thanks.
I will wait:)

Ophir

**Comment by bmartinn** [comment]
Yes, that was a bug :)
It was fixed and will be part of the next release.

If you need, you can test the latest version with:
`$ pip install git+https://github.com/allegroai/trains.git`

**Comment by bmartinn** [comment]
Bug fix is already part of the new release (v0.10.1) please upgrade
`$ pip install -U trains`

Closing issue
